<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>371</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditSearch"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSearch">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>371</width>
      <height>34</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>凉or热</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>凉菜</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>热菜</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>辣不辣</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>辣</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>不辣</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_3">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>食堂</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>食堂</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>学一</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>家园</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="imglabel_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>204</y>
      <width>169</width>
      <height>113</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>3</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>loading</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imglabel_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>204</y>
      <width>169</width>
      <height>113</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>3</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>loading</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imglabel_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>347</y>
      <width>169</width>
      <height>113</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>loading</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imglabel_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>347</y>
      <width>169</width>
      <height>113</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>3</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>loading</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imglabel_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>169</width>
      <height>113</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>3</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>loading</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imglabel_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>169</width>
      <height>113</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>3</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>loading</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="imgButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(52, 152, 219, 0); /* 0.5 is 50% transparency */
    color: white;
    border-radius: 5px;
    padding: 10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="imgButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(52, 152, 219, 0); /* 0.5 is 50% transparency */
    color: white;
    border-radius: 5px;
    padding: 10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="imgButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(52, 152, 219, 0); /* 0.5 is 50% transparency */
    color: white;
    border-radius: 5px;
    padding: 10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="imgButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(52, 152, 219, 0); /* 0.5 is 50% transparency */
    color: white;
    border-radius: 5px;
    padding: 10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="imgButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(52, 152, 219, 0); /* 0.5 is 50% transparency */
    color: white;
    border-radius: 5px;
    padding: 10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="imgButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgba(52, 152, 219, 0); /* 0.5 is 50% transparency */
    color: white;
    border-radius: 5px;
    padding: 10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="RandomButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 200px; /* 将按钮的边框半径设置为100像素，即直径为200 */
    background-color: rgb(155, 4, 4); /* 设置按钮的背景颜色为(155, 4, 4) */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButtonChange">
       <property name="text">
        <string>换一批</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>上传菜品</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>540</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: white; /* 将字体颜色设置为白色 */n}</string>
    </property>
    <property name="text">
     <string>来点推荐</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
